backend/src/main/java/cz/cuni/mff/xrg/odcs/backend/scheduling/Scheduler.java

```diff
@@ -49,37 +48,26 @@
     @Autowired
     private AppConfig appConfig;
 
-    private boolean clusterMode = false;
-
     /**
      * Schedule facade.
      */
     @Autowired
     private ScheduleFacade scheduleFacade;
 
+    /**
+     * Backend instance identifier
+     */
     private String backendID;
 
     @PostConstruct
     private void init() {
-        try {
-            this.clusterMode = this.appConfig.getBoolean(ConfigProperty.BACKEND_CLUSTER_MODE);
-        } catch (MissingConfigPropertyException e) {
-            LOG.info("Running in single mode because cluster mode property is missing in config.properties, {}", e.getLocalizedMessage());
-        }
-        if (this.clusterMode) {
-            this.backendID = this.appConfig.getString(ConfigProperty.BACKEND_ID);
-        }
+        this.backendID = this.appConfig.getString(ConfigProperty.BACKEND_ID);
```

Comment:
Note for tomas-knap: what happens if the config.properties BACKEND_ID is missing?